AUTHORS.rst

```diff
@@ -26,3 +26,4 @@ Please feel free to add your name on this list if you do a PR!
 * Alan Chen (alanchenz)
 * Yixuan Liu (yil039)
 * Gerardo Soto (gcodeon)
+* Paul Luna
```

Comment:
@luna215 can you add your github username in parentheses here? (see the examples above!)